client/src/pages/Signup.js

```diff
@@ -117,31 +155,31 @@ const Signup = () => {
             margin="normal"
             required
             fullWidth
-            name="password1"
+            name="password"
             type="password"
-            id="password1"
+            id="password"
             placeholder="Enter your password"
             onChange={(event) => {
```

Comment:
just noticed this now, but should just pass `handleInputChange` directly 
```
onChange={handleInputChange}
```

Reply:
Fixed